crates/db_views/src/post_view.rs

```diff
@@ -156,6 +165,28 @@ fn queries<'a>() -> Queries<
       .single_value()
   };
 
+  let is_community_followed = |person_id| {
+    community_follower::table
+      .filter(
+        post_aggregates::community_id
+          .eq(community_follower::community_id)
+          .and(community_follower::person_id.eq(person_id)),
+      )
+      .select(community_follower::pending.nullable())
+      .single_value()
+  };
+
+  let number_of_read_comments = |person_id| {
+    person_post_aggregates::table
+      .filter(
+        post_aggregates::post_id
+          .eq(person_post_aggregates::post_id)
+          .and(person_post_aggregates::person_id.eq(person_id)),
+      )
+      .select(person_post_aggregates::read_comments.nullable())
+      .single_value()
+  };
+
```

Comment:
This extra refactoring in post_view.rs should not be done in this PR because it will just result in a more traumatizing merge conflict in my PR in which I'm doing a big refactor of post_view.rs.

Reply:
K, in that case I'll remove everything but the test fixes and creator_is_admin changes.